index.html

```diff
@@ -5,8 +5,199 @@
+                    <div class="p-name">
+                        <h3>Warm-up</h3>
+                    </div>
+                    <div class="p-desc">
+                        <p>
+                            Introduction and Pitching, 
+                            Form teams where need be. 
+                            Building/hacking, 
+                            Feedback, 
+                            Feature requests.
+                        </p>
+                    </div>
+                </article>
+                <article class="program-article">
+                    <div class="program-img">
+                        <img src="https://img.icons8.com/ios/50/FFFFFF/laptop-coding.png"/>                    </div>
+                    <div class="p-name">
+                        <h3> Code</h3>
+                    </div>
+                    <div class="p-desc">
+                        <p>
+                            Code in GitHub/GitLab/BitBucket and deploy in Docker/Automated
+                             CI/Travis CI/Bamboo <br>
+                        </p>
+    
+                    </div>
+                </article>
+                <article class="program-article">
+                    <div class="program-img">
+                        <img src="https://img.icons8.com/ios/50/FFFFFF/widescreen-tv.png"/>
+                    </div>
+                    <div class="p-name">
+                        <h3>Demo</h3>
+                    </div>
+                    <div class="p-desc">
+                        <p>
+                            Teams or individuals will demo what they have hacked for feedback or adoption.
+                        </p>
+                    </div>
+                </article>
+                <article class="program-article">
+                    <div class="program-img">
+                        <img src="https://img.icons8.com/ios/50/FFFFFF/medal2--v1.png"/>                    </div>
+                    <div class="p-name">
+                        <h3>Award</h3>
+                    </div>
+                    <div class="p-desc">
+                        <p>
+                            Teams or individuals will demo what they have hacked for feedback or adoption.
+                        </p>
+                    </div>
+                </article>
+                <article class="program-article">
+                    <div class="program-img">
+                        <img src="https://img.icons8.com/glyph-neue/64/FFFFFF/finish-flag.png"/>                    </div>
+                    <div class="p-name">
+                        <h3>Wrap-up</h3>
+                    </div>
+                    <div class="p-desc">
+                        <p>
+                            Teams or individuals will demo what they have hacked for feedback or adoption.
+                        </p>
```

Comment:
- [ ] Your page should look almost identical to the original design, outside the content differences. You should use the exact same colors, for text, backgrounds, borders, etc, from the original design.

Your design              | Behance design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/43877689/175771809-a8fcde3f-4534-4598-95bf-5974f4fd556e.png)  |  ![image](https://user-images.githubusercontent.com/43877689/175771848-d049daee-8df6-4315-909d-01975f291f10.png)